### PR TITLE
Improve code quality of SLA mechanism in SchedulerJob

### PR DESCRIPTION
**Pull request by turbaszek**
Small changes in `manage_slas` method of SchedulerJob (log messages, f-strings,  if clause simplification) 

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

File: airflow/jobs/scheduler_job.py [hunk 1]
```diff
@@ -453,21 +453,19 @@ def manage_slas(self, dag: DAG, session: Session = None) -> None:
...
+                    self.log.exception("Could not call sla_miss_callback for DAG %s", dag.dag_id)
+            email_content = f"""\
             Here's a list of tasks that missed their SLAs:
             <pre><code>{task_list}\n<code></pre>
             Blocking tasks:
-            <pre><code>{blocking_task_list}\n{bug}<code></pre>
-            """.format(task_list=task_list, blocking_task_list=blocking_task_list,
-                       bug=asciiart.bug)
+            <pre><code>{blocking_task_list}<code></pre>
+            """
```
Comment (turbaszek) [inline airflow/jobs/scheduler_job.py:468]: I'm wondering if we should remove the indentation for this string:
```
In [1]: x = """\
   ...:             Here's a list of tasks that missed their SLAs:
   ...:             <pre><code>{task_list}\n<code></pre>
   ...:             Blocking tasks:
   ...:             <pre><code>{blocking_task_list}\n{bug}<code></pre>
   ...:             """

In [2]:

In [2]: print(x)
            Here's a list of tasks that missed their SLAs:
            <pre><code>{task_list}
<code></pre>
            Blocking tasks:
            <pre><code>{blocking_task_list}
{bug}<code></pre>
```

**Comment by turbaszek** [comment]
@kaxil @potiuk @olchas @TobKed can you please take a look?

**Review by mik-laj — APPROVED** [review]

File: airflow/jobs/scheduler_job.py (hunk 1 above)
Comment (ashb) [reply airflow/jobs/scheduler_job.py:468]: https://docs.python.org/3/library/textwrap.html?highlight=dedent#textwrap.dedent maybe?

Comment (ashb) [reply airflow/jobs/scheduler_job.py:468]: Oh, though this is HTML, so whitespace doesn't really matter.

Comment (turbaszek) [reply airflow/jobs/scheduler_job.py:468]: @ashb yeah that's HTML so indeed it's not a problem as long as the email client can render it

**Pull request merged**